CONTRIBUTING.md

```diff
@@ -53,6 +53,26 @@ We take our open-source community seriously. Please adhere to our [Code of Condu
   - This is run via CI whenever a pull request is opened, or a commit is pushed to a branch that has an open PR.
 - Make sure to cover added code with tests, if it should be tested
 
+### Test Coverage Expectations
+
+To maintain the robustness and reliability of the codebase, we highly value test coverage.
+
+- [Codecov](https://app.codecov.io/github/TBD54566975/dwn-server) is used to track
+  the coverage of our tests and will automatically comment on every pull request
+  showing the impact to overall coverage.
+- We have a strong expectation for every pull request to strive for 100% test
```

Comment:
Let's get rid of this bullet point. It would be nice if contributors add test coverage, but we do not have "strong expectations" of it.